govc/cli/register.go

```diff
@@ -16,11 +16,18 @@ limitations under the License.
 
 package cli
 
+import "os"
+
 var commands = map[string]Command{}
 
 var aliases = map[string]string{}
 
-func Register(name string, c Command) {
+var hideUnreleased = os.Getenv("GOVC_SHOW_UNRELEASED") != "true"
+
+func Register(name string, c Command, unreleased ...bool) {
```

Comment:
Would be good to have a comment here explaining the purpose of unreleased. It's not at all obvious

Reply:
Done